packages/esm-service-queues-app/src/patient-search/patient-scheduled-visits.component.tsx

```diff
@@ -238,15 +235,14 @@ const ScheduledVisits: React.FC<{
                     ) : hasPriority && ind == visitsIndex ? (
                       <ContentSwitcher
                         size="sm"
-                        selectedIndex={1}
+                        selectedIndex={null}
```

Comment:
This fixes a (unrelated, existing) bug that I discovered during testing prevented the user from clicking on whatever the second listed priority is.  So this means that no priority is selected by default, and the user can choose any of the available ones, which results in the form submission.

Reply:
Maybe we can just remove this property altogether?

Reply:
Yeah, I fought this for a while.  That doesn't actually work.  The ContentSwicher component will default this to `0` if not specified, which would make it impossible to select the first element.  Setting it to null explicitly is the only thing that got it to work.  The _right_ solution is not to try to turn a content switcher into a pseudo radio + submit button.  But I'm trying to avoid getting into that here...